src/njoy21/input/ACER/Card1/Npend.hpp

```diff
@@ -0,0 +1,34 @@
+struct Npend : public argument::common::Nin {
+  
+  using Value_t = argument::common::Nin::Value_t;
+  
+  static std::string name(){ return "npend"; }
+  static std::string description(){
+    return
+      "The npend argument specifies the io unit used internally by the\n"
+      "Fortran routine and the filename to be output, e.g. setting npend to\n"
+      "21 or -21 specifies a file named 'tape21' is will be used for the\n"
+      "output\n"
+      "\n"
+      "The signedness of the argument implies the formatting of the file in\n"
+      "question. Positive argument values indicate formatted output i.e.\n"
+      "ascii text files. Negative values indicate unformatted output i.e.\n"
+      "Fortran-style block binary\n"
+      "\n"
+      "npend values are restricted to an absolute value between 20 and 99,\n"
+      "inclusively, and are required to have the same signedness as\n"
+      "(and different absolute value than) the corresponding nendf argument.\n"
+      "\n"
+      "ACE extracts the unionized and linearized cross sections from npend.\n"
+      "npend should have been processed through RECONR. Additionally, if \n"
+      "npend has not been processed through BROADR, the energy grids may not\n"
```

Comment:
This isn't correct.

> ACER uses the information from the ENDF tape mounted on unit
nendf for angle, energy, energy-angle, and photon emission distributions, and
it uses the data on npend for the unionized and linearized cross sections. The
latter file should have been processed through RECONR, and maybe BROADR.
If it wasn't, ACER will still work, but the energy grids may not be quite right.

"If it wasn't" here doesn't refer to running the pendf tape through BROADR. What he's saying is that if the npend tape wasn't run through RECONR (meaning a raw ENDF tape), the energy "may be" incorrect. He says "may be" here---and that might have been true when he wrote it---but with modern tapes, getting a non-sense energy grid is nearly certain